python/src/scipp/coords.py

```diff
@@ -277,5 +299,7 @@ def show_graph(graph, size: str = None):
     :param size: Size forwarded to graphviz, must be a string, "width,height"
                  or "size". In the latter case, the same value is used for
                  both width and height.
+    :param simplified: If ``True``, do not show the conversion functions but
```

Comment:
Minor: Something to this effect
```suggestion
    :param simplified: If ``True``, do not show the conversion functions, only show the potential input and output coordinates.
```
Do not need the "but"